activedirectory/constant.py

```diff
@@ -1,9 +1,10 @@
 """ Copyright start
-  Copyright (C) 2008 - 2021 Fortinet Inc.
+  Copyright (C) 2008 - 2022 Fortinet Inc.
   All rights reserved.
   FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
   Copyright end """
 
+MACRO_LIST = ["User_Enrichment_Playbooks_IRIs"]
```

Comment:
Can we take this out to some global constant (framework level) than each connector. If i am not wrong, this will be used in all enrichment connectors.

Reply:
Yes, We have used these Micro List in every connector but every connector has a different list, that's why we have added in constant.py file